packages/flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: parse_server_sdk_flutter
 description: Flutter plugin for Parse Server, (https://parseplatform.org), (https://back4app.com)
-version: 3.1.3
+version: 3.1.4
```

Comment:
Why is the flutter version being bumped if the change is only in the dart package?

See https://github.com/parse-community/Parse-SDK-Flutter/commit/7fb7ce67d5bea30d583f4b182303ee6a31214185 where we also only had a change in the dart package and only the dart version was bumped.

Reply:
It was my mistake, I thought it was for dart

Reply:
Returned to the previous version